### PR TITLE
Use inline SVG figures in the documents

### PR DESCRIPTION
**Pull request by kimikage**
cf. #359

### CODE REVIEW

**Comment by cormullion** [comment]
Looks terrific! The only thing I'd change (if I was doing it) would be to use spaces instead of commas between the RGB values (partly because I'm a "space after a comma" person (`(r, g, b > (r,g,b)`), which can be frustrating when Julia doesn't do it properly either :)) and because I think it might be easier to read.

Screenshot 2019-12-10 at 09 57 39

I think I used a monospaced font in the old version - these always give good clear spaces.
 
Good to go otherwise!

**Comment by kimikage** [comment]
I agree with you in terms of readability. The reason I didn't use spaces is the ease of mouse (or touch) selection for SVG texts. On Google Chrome, you can select the RGB values as "one" word by double-clicking. However, other browsers do not have such an advantage.

If there is no other opinion, I would like to prioritize the readability.

In addition, I'm going to make some minor fixes regarding the SVG size (or scaling).

**Comment by cormullion** [comment]
I don’t have very strong opinions, so would be happy with any readable solution. I  don’t use Chrome, and  never type RGB values if there’s a simple string name to be used instead. 😂

I’d delete everything unused. Is there a good reason not to?

File: docs/src/namedcolors.md [hunk 1]
```diff
@@ -21,40 +21,86 @@ RGB{N0f8}(0.804,0.361,0.361)
...
+```@example chart
+NamedColorCharts.ColorChartSVG("grays") # hide
+```
+
+!!! info
+    Colors.jl supports the CSS/SVG named colors and the X11 named colors. The
+    CSS/SVG named colors come from the 16 colors defined in HTML3.2 and the X11
+    named colors. There are some unnatural definitions due to the different
+    origins. For example, "LightGray" is lighter than "Gray", but "DarkGray" is
+    also lighter than "Gray".
```
Comment (kimikage) [inline docs/src/namedcolors.md]: @yha, is it okay to close #333?

Comment (yha) [reply docs/src/namedcolors.md]: Yeah, seems fine to me

**Comment by kimikage** [comment]
If you don't like something in docs, please feel free to rewrite it.

**Pull request merged**